src/painter.zig

```diff
@@ -225,8 +226,8 @@ pub fn stroke(
 
 const PaintError = Surface.Error || InternalError || mem.Allocator.Error;
 
-/// Direct paint, writes to surface directly, avoiding compositing. Does not
-/// use AA.
+/// Direct paint, writes to surface directly without AA. Compositing still
+/// happens on a per-stride basis (stack-based).
```

Comment:
TODO remove comments about stack stuff - we were using an FBA while working on this before moving to the batched compositor that does not require any extra memory.

Reply:
I've actually just removed comments from both this and `paintComposte` as neither are public.

Eventually we might be able to consolidate these functions - the line is blurring now between "direct" and indirect paint - the main thing differentiating these two now is the direct method does not do AA and use a mask. It's still significantly faster, but we could possibly just do that another way. No big rush though.